client/node-common/src/service.rs

```diff
@@ -521,10 +521,14 @@ where
             // Here you can check whether the hardware meets your chains' requirements. Putting a link
             // in there and swapping out the requirements for your own are probably a good idea. The
             // requirements for a para-chain are dictated by its relay-chain.
-            if collator && !SUBSTRATE_REFERENCE_HARDWARE.check_hardware(hwbench) {
-                log::warn!(
-                    "⚠️  The hardware does not meet the minimal requirements for role 'Authority'."
+            match SUBSTRATE_REFERENCE_HARDWARE.check_hardware(&hwbench) {
+                Err(err) if collator => {
```

Comment:
Is this intentional? The old code doesn't run `check_hardware` if the node is not a collator. The new code always runs it but only prints the warning if the node is a collator.

Reply:
I noticed that `check_hardware` does not run the benchmarks, it only compares the result. So benchmarks are being run in a function called `gather_hwbench`, which was already being run for all nodes before.

Anyway, I prefer the old version of this piece of code